src/credentials/FabricTable.cpp

```diff
@@ -2164,4 +2164,17 @@ CHIP_ERROR FabricTable::SetFabricIndexForNextAddition(FabricIndex fabricIndex)
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR FabricTable::SetShouldAdvertiseIdentity(FabricIndex fabricIndex, AdvertiseIdentity advertiseIdentity)
+{
+    VerifyOrReturnError(IsValidFabricIndex(fabricIndex), CHIP_ERROR_INVALID_FABRIC_INDEX);
+
+    FabricInfo * fabricInfo  = GetMutableFabricByIndex(fabricIndex);
```

Comment:
```suggestion
    FabricInfo * fabricInfo = GetMutableFabricByIndex(fabricIndex);
```

maybe this was intentional for alignment?

Reply:
It's enforced by restyled.